python/paddle/fluid/tests/unittests/dygraph_to_static/test_break_continue.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+import paddle.fluid as fluid
+from paddle.fluid.dygraph.jit import dygraph_to_static_graph
+
+SEED = 2020
+np.random.seed(SEED)
+
+
+def test_continue_in_for(x):
+    x = fluid.dygraph.to_variable(x)
+    for i in range(10):
+        x += 1
+        if i > 5:
+            continue
+            x += 10086
+        x += i
+    return x
+
+
+def test_continue_in_for_at_end(x):
+    x = fluid.dygraph.to_variable(x)
+    for i in range(10):
+        x += 1
+        if i > 5:
+            continue
+    return x
+
+
+def test_continue_in_while(x):
+    x = fluid.dygraph.to_variable(x)
+    i = fluid.layers.fill_constant(shape=[1], dtype='int32', value=0)
+    while i < 10:
+        i += 1
+        if i > 5:
+            continue
+            x += 10086
+        x += i
```

Comment:
What if `x += 10086` is before `continue` ? Maybe a test that modifying `x` before `continue`/`break` can be added.

Reply:
Sure, I added a test case before continue/break. Please see test_break_continue_in_for